tool/doc.dart

```diff
@@ -34,10 +34,15 @@ rules are considered stable, while others may be marked **experimental**
 to indicate that they are under review.
 
 Rules can be selectively enabled in the analyzer using
-[analysis options](https://pub.dartlang.org/packages/analyzer).  An
-auto-generated list enabling all options is provided
-[here](options/options.html).  As some lints may contradict each other, only
-some lints will be enabled in practice, but this list should provide a
+[analysis options](https://pub.dartlang.org/packages/analyzer)
+or through the
+[.analysis-options file](https://www.dartlang.org/guides/language/analysis-options#the-analysis-options-file). 
```

Comment:
shouldn't that be `analysis_options.yaml`?
From what I got this is now the preferred name.